packages/conf-xkbcommon/conf-xkbcommon.1/opam

```diff
@@ -9,15 +9,18 @@ depends: [
   "conf-pkg-config" {build}
 ]
 depexts: [
-  ["libxkbcommon-dev"] {os-family = "debian"}
-  ["libxkbcommon-devel"] {os-distribution = "fedora"}
-  ["libxkbcommon-devel"] {os-distribution = "rhel"}
-  ["libxkbcommon-devel"] {os-distribution = "centos"}
-  ["libxkbcommon-devel"] {os-family = "suse"}
-  ["libxkbcommon-dev"] {os-distribution = "alpine"}
-  ["libxkbcommon"] {os-distribution = "arch"}
-  ["libxkbcommon"] {os = "freebsd"}
+  ["libxkbcommon-devel"] {os = "cygwin" | os-distribution = "cygwinports"}
+  ["libxkbcommon-dev"] {os-family = "debian" | os-family = "ubuntu"}
+  ["libxkbcommon-devel"] {os-family = "fedora" | os-family = "rhel" | os-family = "centos"}
+  ["libxkbcommon-devel"] {os-family = "mandriva" | os-family = "openmandriva" | os-family = "mageia"}
+  ["libxkbcommon-devel"] {os-family = "suse" | os-family = "opensuse" | os-family = "sles"}
+  ["libxkbcommon-dev"] {os-family = "alpine"}
+  ["libxkbcommon"] {os-family = "arch" | os-family = "archlinux"}
+  ["x11-libs/libxkbcommon"] {os-family = "gentoo"}
+  ["libxkbcommon"] {os-family = "bsd" & os != "openbsd"}
+  ["xkbcommon"] {os = "openbsd"}
```

Comment:
i prefer be more explicit about what is supported (white-list vs black-list) in this case
```suggestion
  ["libxkbcommon"] {os = "freebsd" | os = "dragonfly" | os = "netbsd"}
  ["xkbcommon"] {os = "openbsd"}
```